inspirehep/modules/disambiguation/beard.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Inspire.
+# Copyright (C) 2016 CERN.
+#
+# Inspire is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Inspire is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Inspire; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Communication handler with Beard Celery service."""
+
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+import celery
+from flask import current_app
+
+
+def make_beard_clusters(records, signatures):
+    """Dispatch a clustering task to Beard Celery instance.
+
+    The method receives a list of records and a list of signatures representing
+    records that will be clustered by Beard algorithm.
+
+    The argument 'records' is a list of dictionaries containing values like
+    authors of a particular record, earliest date recorded of the publication
+    and finally publication UUID.
```

Comment:
@jacquerie I think `record` in this context is to be considered as a `beard_record`. @jacenkow indeed what are the keys that are expected? Is there a formal description to which you can refer in this description?

E.g.  to say: "see docstring of function `foo` for a description of the structure of a record as expected by beard."